src/test/java/org/folio/linked/data/e2e/ResourceControllerFolioIT.java

```diff
@@ -56,7 +56,9 @@ protected void checkSearchIndexMessage(Long id, ResourceIndexEventType eventType
   @Override
   protected void checkInventoryMessage(Long id, InstanceIngressEvent.EventTypeEnum eventType) {
     awaitAndAssert(() ->
-      assertTrue(inventoryTopicListener.getMessages().stream().anyMatch(m -> m.contains(id.toString())
+      assertTrue(inventoryTopicListener.getMessages()
+        .stream()
+        .anyMatch(m -> m.contains(id.toString())
```

Comment:
what's the reason of such changes? Especially, leaving the same wording at line 51-52? :D

Reply:
It was accidentally xD , 
reverted